example/lib/main.dart

```diff
@@ -21,7 +21,19 @@ import 'edgeIdentity.dart';
 import 'edgebridge.dart';
 import 'userprofile.dart';
 
-void main() {
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  //initialize AEP SDK
+  MobileCore.setLogLevel(LogLevel.trace);
+  InitOptions initOptions = InitOptions(
+    appId: "YOUR_APP_ID",
+    lifecycleAutomaticTracking: false,
+    lifecycleAdditionalContextData: {"key": "value"},
+    appGroupIOS: "group.com.example",
+  );
+
+  await MobileCore.initialize(initOptions: initOptions);
```

Comment:
This is incorrect, as it will delay the app load until the SDK initializes. Could you move this to a more relevant place?